src/Access/AccessControl.cpp

```diff
@@ -447,23 +450,35 @@ void AccessControl::setCustomSettingsPrefixes(const String & comma_separated_pre
     setCustomSettingsPrefixes(prefixes);
 }
 
-void AccessControl::setPlaintextPasswordSetting(bool allow_plaintext_password_)
+bool AccessControl::isSettingNameAllowed(const std::string_view & setting_name) const
```

Comment:
```suggestion
bool AccessControl::isSettingNameAllowed(const std::string_view setting_name) const
```
`std::string_view` is just pointer and size so it's better to copy them than add an extra indirection with reference.